arm-sql/2014-04-01/swagger/geoBackupPolicies.json

```diff
@@ -0,0 +1,307 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database",
+    "description": "Provides read and update functionality for Azure SQL Database geo backup policies.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/geoBackupPolicies/{geoBackupPolicyName}": {
+      "put": {
+        "tags": [
+          "GeoBackupPolicies"
+        ],
+        "operationId": "GeoBackupPolicies_CreateOrUpdate",
+        "description": "Updates a database geo backup policy.",
+        "x-ms-examples": {
+          "Update geo backup policy": { "$ref": "../examples/GeoBackupPoliciesCreateOrUpdate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database."
+          },
+          {
+            "name": "geoBackupPolicyName",
```

Comment:
Since this is singleton, should we go with Databases_CreateOrUpdateGeoBackupPolicy operation id instead?